src/applications/vaos/new-appointment/components/DateTimeRequestPage.jsx

```diff
@@ -35,133 +32,150 @@ export function getOptionsByDate() {
-    const calendarData = data?.calendarData || {};
-    const { currentlySelectedDate, selectedDates } = calendarData;
-    const { validationError } = this.state;
-
-    const additionalOptions = {
-      fieldName: 'optionTime',
-      required: true,
-      maxSelections: 2,
-      validationMessage:
-        'Please select a preferred time or unselect this date to continue',
-      getOptionsByDate,
-    };
-
-    return (
-      <div className="vaos-form__detailed-radio">
-        <h1 className="vads-u-font-size--h2">{pageTitle}</h1>
-        <p>
-          You can choose up to 3 dates. A scheduling coordinator will call you
-          to schedule the best time for your appointment.
-        </p>
-        <CalendarWidget
-          monthsToShowAtOnce={2}
-          multiSelect
-          maxSelections={maxSelections}
-          onChange={newData => {
-            this.validate(newData);
-            this.props.onCalendarChange(newData);
-          }}
-          minDate={moment()
-            .add(5, 'days')
-            .format('YYYY-MM-DD')}
-          maxDate={moment()
-            .add(120, 'days')
-            .format('YYYY-MM-DD')}
-          currentlySelectedDate={currentlySelectedDate}
-          selectedDates={selectedDates}
-          selectedIndicatorType={CALENDAR_INDICATOR_TYPES.BUBBLES}
-          additionalOptions={additionalOptions}
-          validationError={
-            this.state.submitted || this.isMaxSelectionsError()
-              ? validationError
-              : null
-          }
-        />
-        <FormButtons
-          onBack={this.goBack}
-          onSubmit={this.goForward}
-          pageChangeInProgress={pageChangeInProgress}
-          loadingText="Page change in progress"
-        />
-      </div>
-    );
-  }
+  return (
+    <div className="vaos-form__detailed-radio">
+      <h1 className="vads-u-font-size--h2">{pageTitle}</h1>
+      <p>
+        You can choose up to 3 dates. A scheduling coordinator will call you to
+        schedule the best time for your appointment.
+      </p>
+      <CalendarWidget
+        monthsToShowAtOnce={2}
+        multiSelect
+        maxSelections={maxSelections}
+        onChange={newData => {
+          validate({ calendarData: newData, setValidationError });
+          onCalendarChange(newData);
+        }}
+        minDate={moment()
+          .add(5, 'days')
+          .format('YYYY-MM-DD')}
+        maxDate={moment()
+          .add(120, 'days')
+          .format('YYYY-MM-DD')}
+        currentlySelectedDate={currentlySelectedDate}
+        selectedDates={selectedDates}
+        selectedIndicatorType={CALENDAR_INDICATOR_TYPES.BUBBLES}
+        additionalOptions={additionalOptions}
+        validationError={
+          submitted || isMaxSelectionsError(validationError)
+            ? validationError
+            : null
+        }
+      />
+      <FormButtons
+        onBack={() => goBack({ routeToPreviousAppointmentPage, history })}
```

Comment:
Since `goBack()` is just a 1 line function, you should just be able to replace this line with:

`onBack={() => routeToPreviousAppointmentPage(history, pageKey)}`